fleet/lib/openvpn-db/overlays/ayekan/mariadb-cr.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: k8s.mariadb.com/v1alpha1
+kind: MariaDB
+metadata:
+  name: mariadb
+  namespace: openvpn-db
+spec:
+  replicas: 3
+  galera:
+    enabled: true
+    primary:
+      podIndex: 0
+      automaticFailover: true
+    sst: mariabackup
+
+  rootPasswordSecretKeyRef:
+    name: mariadb
+    key: mariadb-root-password
+  username: mariadb
+  passwordSecretKeyRef:
+    name: mariadb
+    key: mariadb-root-password
+  database: mariadb
+
+  image: docker.io/library/mariadb:11.4.5
+  imagePullPolicy: IfNotPresent
+  imagePullSecrets:
+    - name: dockerhub-secret
+
+  storage:
+    size: 100Gi
+    storageClassName: rook-ceph-block
+
+  connection:
+    secretName: connection-mariadb
+    secretTemplate:
+      key: dsn
+    healthCheck:
+      interval: 10s
+      retryInterval: 3s
+    params:
+      parseTime: "true"
+
+  myCnf: |
+    [mariadb]
+    bind-address=*
+    skip-name-resolve
+    connect_timeout=30
+    wait_timeout=28800
+    interactive_timeout=28800
+    net_read_timeout=60
+    net_write_timeout=60
+    max_connections=500
+    max_allowed_packet=64M
+    thread_cache_size=100
+    table_open_cache=2048
+    open_files_limit=65535
+    default_storage_engine=InnoDB
+    binlog_format=row
+    innodb_autoinc_lock_mode=2
+    innodb_buffer_pool_size=1024M
+
+  livenessProbe:
```

Comment:
Is there no default liveness/readiness probe?

Reply:
I checked the MariaDB Operator source (GitHub) and did not find any injected default liveness/readiness probes. I added these probes to check that the  MariaDB process is responsive and restart it if it's not. The other probe ensures the node is not just “running” but also in sync with the Galera cluster.